rosbag2_storage_mcap/test/rosbag2_storage_mcap/test_mcap_storage.cpp

```diff
@@ -200,3 +203,40 @@ TEST_F(TemporaryDirectoryFixture, can_write_mcap_with_zstd_configured_from_yaml)
   }
 }
 #endif  // #ifdef ROSBAG2_STORAGE_MCAP_HAS_STORAGE_OPTIONS
+
+TEST_F(TemporaryDirectoryFixture, mcap_contains_ros_distro)
+{
+  // const auto expected_file = rcpputils::fs::path(temporary_dir_path_) / "rosdistro_bag.mcap";
+  const auto expected_file = rcpputils::fs::path("rosdistro_bag.mcap");
```

Comment:
It looks like it is supposed to be a path to the temporary folder but was forgotten to change back after debugging.